### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request: CONFLICT #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-python-library.git`](https://github.com/cisagov/skeleton-python-library.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Take ownership of this pull request by removing any other assignees.

1. Clone the repository locally, and reapply the merge:

    ```console
    git clone git@github.com:cisagov/github-label-management.git github-label-management
    cd github-label-management
    git remote add skeleton https://github.com/cisagov/skeleton-python-library.git
    git remote set-url --push skeleton no_push
    git switch develop
    git checkout -b lineage/skeleton --track origin/develop
    git pull skeleton HEAD
    git status
    ```

1. Review the changes displayed by the `status` command.  Fix any conflicts and
   possibly incorrect auto-merges.

1. After resolving each of the conflicts, `add` your changes to the
   branch, `commit`, and `push` your changes:

    ```console
    git add setup.py 
    git commit
    git push --force --set-upstream origin lineage/skeleton
    ```

    Note that you may _append_ to the default merge commit message
    that git creates for you, but *please do not delete the existing
    content*.  It provides useful information about the merge that is
    being performed.

1. Wait for all the automated tests to pass.

1. Check the "Everything is cool" checkbox below:

    - [x] ✌️ The conflicts in this pull request have been resolved.

1. Mark this draft pull request "Ready for review".

------------

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -45,16 +45,20 @@ def get_version(version_file):
     name="example",
     # Versions should comply with PEP440
     version=get_version("src/example/_version.py"),
-    description="Example python library",
+    description="Example Python library",
```
Comment (jsf9k) [inline setup.py:48]: Can you update the description?

**Review by jsf9k — CHANGES_REQUESTED** [review]
THese changes look great, except that the `description` field in `setup.py` should be changed.

**Comment by dav3r** [comment]
I reassigned ownership of this PR to @hillaryj since she mentioned that she was planning to make some updates this repo.  Please ensure that the:
* README in this repo is updated from the skeleton to include relevant info
* Example Python code in this repo is either modified for this repo or removed entirely (if it's not needed)

**Comment by hillaryj** [comment]
I have a branch for `initial-commits` that I'm pushing changes to - it's currently in-progress. Should we merge the skeleton updates into the base for now until the `initial-commits` PR is ready?

File: setup.py (hunk 1 above)
Comment (hillaryj) [reply setup.py:48]: The updated description is in the `initial-commits` branch, which is still in progress but can be seen here: https://github.com/cisagov/github-label-management/blob/initial-commits/setup.py

**Comment by dav3r** [comment]
> I have a branch for `initial-commits` that I'm pushing changes to - it's currently in-progress. Should we merge the skeleton updates into the base for now until the `initial-commits` PR is ready?

I'm fine with that approach. 👍

**Review by dav3r — APPROVED** [review]
👍 👍

**Review by jsf9k — APPROVED** [review]
I'm onboard with merging these changes into the `develop` branch while @hillaryj continues working on her `initial-commits` branch.

File: setup.py (hunk 1 above)
Comment (jsf9k) [reply setup.py:48]: Got it.

**Comment by jsf9k** [comment]
@hillaryj, go ahead and merge at your leisure.

**Pull request merged**